frontend/src/components/common/SectionHeader.tsx

```diff
@@ -28,6 +29,7 @@ export interface SectionHeaderProps {
   actions?: React.ReactNode[] | null;
   noPadding?: boolean;
   headerStyle?: HeaderStyle;
+  actionsBesideTitle?: React.ReactNode[];
```

Comment:
Let's call this `titleSideActions` please.

Reply:
updated